pages/features.html

```diff
@@ -6,92 +6,122 @@
+      <p><b>{% trans "Programmable Mapping Engine" %}</b><br>
+        {% blocktrans with midi_scripting_wiki_link="/wiki/doku.php/midi_scripting" %}Cutting-edge JavaScript-based scripting engine provides maximum flexibility. MIDI or HID messages can be mapped onto custom functions which execute complex behaviour. <a href="{{ midi_scripting_wiki_link }}">Read more</a>{% endblocktrans %}
+      </p>
+    </div>
+    <div class="halfbox_right">
+      <div style="height: 200px;">
+        <center><img src="{% static '/static/images/feature_vinylcontrol.png' %}"></center>
+      </div>
+
+      <h2>{% trans "Free Timecode Vinyl Control" %}</h2>
+      <p>
+        {% trans "Control your digital music files from turntables or CDJs and a mixer using timecoded vinyl or CDs. Mixxx is the only free timecode vinyl control software for Windows, Mac OS X, and Linux." %}
+      </p>
+
+      <p>
+        {% blocktrans with vinyl_control_manual_link="/manual/latest/chapters/vinyl_control.html" %}Serato CV 2, Serato CV 2.5, Traktor Scratch Mk1, and Mixvibes DVS V2 timecode are supported with absolute positioning, needle cueing, and end-of-record detection. Use any sound card(s) your OS supports to read timecode signals and play your music in high quality. <a href="{{ vinyl_control_manual_link }}">Read more</a>{% endblocktrans %}
+      </p>
+    </div>
+
+  <div style="clear: both;"></div>
+
+    <h1>{% trans "Sophisticated Music Library" %}</h1>
     <p>
-      {% trans "The Mixxx music library was designed from the ground up for DJs. Powered by a high-performance database, accessing and organizing your music is easy and intuitive." %}
+      {% trans "Stay organized and find the music you're looking for the moment you need it. Powered by a high-performance database, navigation and searching your collection is intuitive and fast." %}
     </p>
 
     <div style="clear: both;"></div>
 
     <div class="halfbox_left">
-      <H2><img src="{% static '/static/images/ic_library_itunes.png' %}" class="feature_icon">{% trans "Organization" %}</H2>
+      <h2><img src="{% static '/static/images/ic_library_itunes.png' %}" class="feature_icon">{% trans "Organization" %}</h2>
       <p class="feature_indent">
         <b>{% trans "Crates and Playlists" %}</b><br>
-        {% trans "Sort your music the way you want." %}
+        {% trans "Sort your music the way you want. Use playlists to arrange plans for a set and unordered crates to build your own categorization system." %}
       </p>
       <p class="feature_indent">
-        <b>{% trans "Disk Browsing" %}</b><br>
-        {% trans "Need to find music outside your library? Browse and load tunes from any hard disk." %}
+        <b>{% trans "Search &amp; Sort" %}</b><br>
+        {% trans "Filter your collection by specific tags to narrow your choices or just start typing to find the track you're thinking of. Sort the library table hierarchically with multiple tags at a time." %}
```

Comment:
go through and remove excess incidences of the word "just" -- it's lazy and I do it too, but it sounds cheap